data/transition-sites/phe_hpa.yml

```diff
@@ -9,3 +9,4 @@ aliases:
 - hpa.org.uk
 - www.hpa.nhs.uk
 - hpa.nhs.uk
+- legacytools.hpa.org.uk
```

Comment:
No, it doesn't. So this should be setup as a configured site for Public Health England. Apologies for the lack of detail.